src/main/resources/templates/swift/xcode/test/${packageNameFolder}Tests/${packageNameFile}Tests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+@testable import ${packageName}
+
+class ${packageName}Tests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        try super.setUpWithError()
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        try super.tearDownWithError()
+    }
+
+    func testSampleText() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        let sample = SampleText()
+        sample.text = "My Artemis Test"
```

Comment:
I would make it consistent to the template solution:

```suggestion
        let sample = SampleText("My Artemis Text")
```

OR use the setText()-function

```suggestion
        let sample = SampleText()
        sample.setText("My Artemis Test")
```

Reply:
the troublesome part here is the template. If the template runs and the `init()` or `setText()` functions are not implemented yet, the Test would fail and throw a runtime exception.. 

This is the only way I could think of to solve this problem. I don't want to pre-fill the template code with either, because the student should solve that. 

Do you have a better idea?

Reply:
Yes, that's true. I have not thought about it. I also think this is the only way to do it.